app/src/main/java/com/jerboa/ui/components/common/InputFields.kt

```diff
@@ -680,15 +677,13 @@ fun MyMarkdownText(
     markdown: String,
     color: Color = MaterialTheme.colorScheme.onSurface,
     onClick: () -> Unit,
+    onLongClick: (() -> Unit)? = null,
```

Comment:
Make sure this is ` -> Unit` with no default, just like the one above it.

Reply:
onLongClick was never available before. If  this parameter becomes non optional then i must update all `CreateMarkdownView()` calls with an additional unneeded empty object.

Both parameters should probably have a default of `(() -> Unit)? = null` as click events on the ultimately rendered TextView should be optional in any case without needing redundant `{}, {}` everywhere. An example could be a markdown rendered popup.